javascript/features/announce/announce_manager.js

```diff
@@ -62,18 +66,42 @@ class AnnounceManager {
     // Announces |message| to all in-game administrators. Optionally |args| may be passed if
     // |message| is an instance of Message, which is common infrastructure for user-visible text.
     announceToAdministrators(message, ...args) {
+        this.announceToAdministratorsWithFilter(message, PlayerSetting.ANNOUNCEMENT.UNCATEGORIZED, PlayerSetting.SUBCOMMAND.GENERAL, args);
```

Comment:
nit: here and elsewhere, please stick to a maximum line length of 100 characters.

Do you use Visual Studio Code as your editor? We can probably check in a .vscode configuration file to make these things more obvious :).

Reply:
Yes, I do use VS code.

The really long lines are really just an error on my end, I'm aware of it. 
However it would be nice to have it configured.